common/src/main/java/com/microsoft/identity/common/crypto/AndroidAuthSdkStorageEncryptionManager.java

```diff
@@ -81,6 +81,8 @@ public AndroidAuthSdkStorageEncryptionManager(@NonNull final Context context,
         if (PredefinedKeyLoader.USER_PROVIDED_KEY_IDENTIFIER.equalsIgnoreCase(keyIdentifier)) {
             if (mPredefinedKeyLoader != null) {
                 return Collections.<AbstractSecretKeyLoader>singletonList(mPredefinedKeyLoader);
+            } else if (AuthenticationSettings.INSTANCE.getShouldIgnorePredefinedKeyLoaderNotFoundError()) {
```

Comment:
If the deta is unencrypted, it'll likely go into line 95.

Reply:
(My understanding of the issue is that Office gets the unencrypted data while it's reading).

Reply:
reverted this change back to what it is in dev, except for the additional logging